### PR TITLE
Edit: Markdown-Images

### LINKED ISSUE #4891: [Edit] Markdown: Images
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like to edit the `Images` concept entry under Markdown. The entry lives in `content/markdown/concepts/images/images.md`.

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Edit the `Linked Images` section to add the output of the Markdown example.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by danitellini**
### Description

Topic: Edited Markdown: Images file
Files Edited: `docs/content/markdown/concepts/images/images.md`
Files Added: `docs/media/codey.jpg`

I wasn't able to use the original example as it was, though I left it the same. The "output" path is different to compensate, and I added the only image I could find of Codey (drinking bubble tea) so I adjusted the titles in the example as well.

### Issue Solved

Closes #4891 

### Type of Change

- ~Adding a new entry~
- Editing an existing entry (fixing a typo, bug, issues, etc)
- ~Updating the documentation~

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing to Codecademy Docs @danitellini  😄 

The entry looks good for a second round of review! 🚀

**Comment by mamtawardhani** [comment]
@Sriparno08 assigning this PR to you for the second review so that we can close this on priority, thank you! 😄

File: content/markdown/concepts/images/images.md
Comment (danitellini) [inline content/markdown/concepts/images/images.md:53]: The example is a linked image, so shouldn't the link work?

Comment (Sriparno08) [reply content/markdown/concepts/images/images.md:53]: Hey @danitellini, the link will work after the PR is merged.

**Review by Sriparno08 — APPROVED** [review]
Looks good, @danitellini!

**Pull request merged**
